src/couch_replicator_httpc_pool.erl

```diff
@@ -147,9 +147,13 @@ code_change(_OldVsn, #state{}=State, _Extra) ->
     {ok, State}.
 
 
-terminate(_Reason, State) ->
-    lists:foreach(fun ibrowse_http_client:stop/1, State#state.free),
-    lists:foreach(fun ibrowse_http_client:stop/1, State#state.busy).
+terminate(_Reason, #state{free=Free, busy=Busy}) ->
+    [begin
+        catch unlink(Pid),
```

Comment:
Good point. Copied that code from a place where Pid could have been nil, then it would have crashed. Don't think it can be nil here. Will fix.